### DISCUSSION TITLE
[RFC] Adding Support for Android 13

### DISCUSSION THREAD

**Discussion by bkabbott** [zeapo/Android-Password-Store#2664] [Q&A] [answered]
I would like to add support for Android 13, which is what my phone uses.

I've cloned the repository and I am getting the error message:

`The project is using an incompatible version (AGP 8.3.0-alpha01) of the Android Gradle plugin. Latest supported version is AGP 8.1.0
`
When I run 

`gradle -v

------------------------------------------------------------
Gradle 8.3
------------------------------------------------------------

Build time:   2023-08-17 07:06:47 UTC
Revision:     8afbf24b469158b714b36e84c6f4d4976c86fcd5

Kotlin:       1.9.0
Groovy:       3.0.17
Ant:          Apache Ant(TM) version 1.10.13 compiled on January 4 2023
JVM:          11.0.20 (Ubuntu 11.0.20+8-post-Ubuntu-1ubuntu122.04)
OS:           Linux 5.15.0-25-generic amd64

`

I am running Android Studio Giraffe on Ubuntu 22.04 (2022.3.1).

Any help getting this to run would be appreciated with any guidelines on this pull request I would like to submit.

**Comment by msfjarvis** [comment, marked as answer]
The contribution guide mentions that APS uses pre-release versions of Android Studio at the moment, so you'll need Android Studio Iguana. Either way, Android 13 has been supported since its release. A quick search of the issue tracker would've revealed multiple similar questions such as https://github.com/android-password-store/Android-Password-Store/issues/2644, https://github.com/android-password-store/Android-Password-Store/issues/2587, https://github.com/android-password-store/Android-Password-Store/issues/2553, and https://github.com/android-password-store/Android-Password-Store/issues/2417.
